src/app/todos/todo-list/todo-list.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { Subscription } from 'rxjs';
+
+import { TodoService } from '../todo.service';
+import { Todo, TodoStatus } from '../todo.model';
+
+@Component({
+  selector: 'app-todo-list',
+  templateUrl: './todo-list.component.html',
+  styleUrls: ['./todo-list.component.scss']
+})
+export class TodoListComponent implements OnInit, OnDestroy {
+  todos: Todo[] = [];
+  // @ts-ignore
+  subscription: Subscription;
+  // @ts-ignore
+  titleForm: FormGroup;
+  editingTodoId: string | null = null;
+  readonly TodoStatus = TodoStatus;
+
+  constructor(private todoService: TodoService) {}
+
+  ngOnInit() {
+    this.initForm();
+    this.subscription = this.todoService.todosChanged
```

Comment:
this.subscription is not really a name that tell me anything about it. A variable name should be speaking about itself